ui/web-v2/apps/admin/src/pages/apiKey/index.tsx

```diff
@@ -271,6 +271,8 @@ export const APIKeyIndexPage: FC = memo(() => {
         createAPIKey({
           environmentNamespace: currentEnvironment.id,
           name: data.name,
+          // TODO: Implement the `role` radio button in the create page
+          role: 1, // SDK_CLIENT
```

Comment:
This value will be selected in the create page in another PR when I implement the API for the Server SDK.